examples/complex-transaction/create-ship.sh

```
@@ -0,0 +1,78 @@
+  --data '{"query":"query { nextShipTokenName(spacetimeAddress: \"addr_test1wzmvtc20xxhseyj3tns4vcj6l3r5nccvamch3nawr7ffllcmwmxeq\", spacetimePolicyId: \"b6c5e14f31af0c92515ce156625afc4749e30ceef178cfae1f929fff\") { shipName pilotName } }"}')
+
+SHIP_NAME=$(echo $NEXT_SHIP_RES | jq -r '.data.nextShipTokenName.shipName')
+PILOT_NAME=$(echo $NEXT_SHIP_RES | jq -r '.data.nextShipTokenName.pilotName')
+
+SHIP_NAME=$(printf '%s' "$SHIP_NAME" | xxd -p -u)
+PILOT_NAME=$(printf '%s' "$PILOT_NAME" | xxd -p -u)
+
+LAST_SLOT_RES=$(curl --location 'https://8000-skillful-employee-kb9ou6.us1.demeter.run/graphql' --header 'Content-Type: application/json' \
+  --data '{"query":"query { lastSlot { slot } }"}')
+
+TIP_SLOT=`expr $(echo $LAST_SLOT_RES | jq -r '.data.lastSlot.slot') + 300`
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)